### PR TITLE
Add BBVA AI Factory to the Website - Stories Page

### PR DESCRIPTION
**Pull request by rahermur**
I work for the BBVA AI Factory and know they we are doing InnerSource through the Mercury project

### CODE REVIEW

File: content/en/stories/_index.md [hunk 1]
```diff
@@ -44,6 +44,9 @@ aliases:
       {{< company name="BBC" image="/images/logos/bbc.png" video="https://www.youtube.com/watch?v=pEGMxe6xz-0" author_name="Steph Egan" author_title="Software Engineering Team Lead" >}}
       Our tools have been built been built from the ground up with an expectation of InnerSource … The way that we encourage that in a very disparate organization is mostly word of mouth and clear documentation.
       {{< /company >}}
+      {{< company name="BBVA AI Factory" image="/images/logos/BBVA_AI_factory.png" article="https://www.bbvaaifactory.com/mercury-acelerando-la-reutilizacion-en-ciencia-de-datos-dentro-de-bbva/" author_name="Rafael Hernandez Murcia" author_title="Senior Manager Data Scientist" >}}
+      Mercury is a collaborative Data Science library in Python that applies the principles of Inner source at BBVA.
```
Comment (spier) [inline content/en/stories/_index.md]:
```suggestion
      Mercury is a collaborative Data Science library in Python that applies the principles of InnerSource at BBVA.
```

Changing the spelling to how we write it within the InnerSource Commons ;)

**Review by spier — COMMENTED** [review]
Thanks for adding yourself to the stories page!

If you check out the existing [stories](https://innersourcecommons.org/stories/), you will notice that most orgs describe in their quote how they are using InnerSource to overcome a specific engineering challenge that they had.

Therefore I am wondering if you could talk about Mercury more indirectly, as the readers would not know what Mercury is?

i.e. what was the challenge that you had prior to the project, and how did the InnerSource approach that you took with Mercury help you to overcome that challenge?

**Comment by rahermur** [comment]
Got it. Totally makes sense.  I just changed the quote to focus on code reusability in the data science domain.

**Review by spier — APPROVED** [review]

**Review by spier — APPROVED** [review]
@rahermur this looks great now, happy to merge this.

Only one question:
For your role title, is "Senior Manager Data **Scientist**" correct, or should it be "Science" as the last word?

**Comment by rahermur** [comment]
Yes, correct. I prefer "Senior Manager Data Scientist" or "Senior Manager - Data Science"

**Comment by spier** [comment]
Perfect, let's keep it as is then!

**Comment by spier** [comment]
@rahermur your logo+story is now live at https://innersourcecommons.org/stories/ 🚀

**Comment by rahermur** [comment]
Awesome! Thanks

**Pull request merged**